src/screens/DiscoveryScreenV2/index.js

```diff
@@ -3,6 +3,7 @@ import axios from 'axios';
 import {Keyboard, Platform, ScrollView, StatusBar, StyleSheet} from 'react-native';
 import {SafeAreaView} from 'react-native-safe-area-context';
 
+import {useNavigation} from '@react-navigation/core';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `useNavigation` hook from `@react-navigation/core` is imported but the old hunk doesn't show it being used anywhere. If it's not used, it should be removed to keep the code clean.
```diff
- import {useNavigation} from '@react-navigation/core';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->